### DISCUSSION TITLE
Bring to front for arrange applications

### DISCUSSION THREAD

**Discussion by tomzag** [rxhanson/RectanglePro-Community#58] [Ideas]
I'm happy user of Rectangle Pro.

What would make my life even easier is an option "Bring to front" in the "Applications" tab for a group called "Bring to front".

<img width="412" alt="Bildschirmfoto 2022-04-27 um 11 12 08" src="https://user-images.githubusercontent.com/12913617/165485258-059cbed8-ca8f-4f53-8ccb-0468490501d2.png">

So on pressing the shortcut it not only should order the applications but also bring them to front if they are not.

Thomas

**Comment by rxhanson** [comment]
Excellent idea. I will mark this for one of the next releases.
